projects/shared-modules/hms-video/components/hms-video/conference/conference-settings/conference-settings.component.ts

```diff
@@ -41,10 +41,13 @@ export class ConferenceSettingsComponent implements OnInit, OnDestroy {
   }
 
   getMediaPermissions(): void {
+    const microphone = 'microphone' as PermissionName;
+    const camera = 'camera' as PermissionName;
+
     this.subscriptions.push(
       combineLatest(
-        this.localMediaService.getMediaPermissions('camera'),
-        this.localMediaService.getMediaPermissions('microphone'),
+        this.localMediaService.getMediaPermissions(camera),
+        this.localMediaService.getMediaPermissions(microphone),
```

Comment:
@ArshdeepGrover please ensure this works perfectly

Reply:
Working proper!!